src/System.Data.Odbc/tests/Configurations.props

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <BuildConfigurations>
```

Comment:
Yes that line is https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/src/System.Data.SqlClient.csproj#L8 is dead code as well as it only has a netstandard configuration now. @saurabh500 once you want that library to support .NET Framework (assuming you do) then you will need to add the netfx configuration to it.